src/applications/vaos/tests/list/cc-appointments.unit.spec.jsx

```diff
@@ -45,7 +44,6 @@ describe('VAOS integration: upcoming CC appointments', () => {
       queryByText,
     } = renderWithStoreAndRouter(<FutureAppointmentsList />, {
       initialState,
-      reducers,
```

Comment:
Our reducers are all included by the helper we're using, so this isn't necessary